### PR TITLE
adds select all functionality to professional interests

### PR DESCRIPTION
**Pull request by alex-418**
Completes https://github.com/BCcampus/eypd/issues/396

### CODE REVIEW

File: assets/scripts/cwp-notify-categories.js [hunk 1]
```diff
@@ -8,6 +8,11 @@
 
     $(document).ready(function () {
 
+        // Handles the select all functionality
+        $('#select_all').click(function() {
+            $(':checkbox').prop('checked', this.checked);
```
Comment (bdolor) [inline assets/scripts/cwp-notify-categories.js]: this doesn't work. 
change to `$('.notifiable-categories').prop('checked', this.checked);` and it works

Comment (alex-418) [reply assets/scripts/cwp-notify-categories.js]: In chrome you'll have to clear cache, then it should work. If not let me know.

Comment (bdolor) [reply assets/scripts/cwp-notify-categories.js]: without specifying a class that belongs to the checkbox we're targeting you hit all checkboxes on the page (including ones we don't want to target)

Comment (alex-418) [reply assets/scripts/cwp-notify-categories.js]: oh, I thought you meant it didn't work at all. Okay, that makes perfect sense I'll update it.

**Pull request merged**